src/SFA.DAS.AssessorService.Application/Handlers/Staff/StaffSearchHandler.cs

```diff
@@ -142,16 +143,16 @@ private List<StaffSearchItems> MatchUpExistingCompletedStandards(List<StaffSearc
             return searchResults;
         }
 
-        private List<StaffSearchItems> PopulateStandards(List<StaffSearchItems> searchResults, IAssessmentOrgsApiClient assessmentOrgsApiClient, ILogger<SearchHandler> logger)
+        private List<StaffSearchItems> PopulateStandards(List<StaffSearchItems> searchResults, IStandardService standardService, ILogger<SearchHandler> logger)
         {
-            var allStandards = assessmentOrgsApiClient.GetAllStandards().Result;
+            var allStandards = standardService.GetAllStandards().Result;
 
             foreach (var searchResult in searchResults)
             {
                 var standard = allStandards.SingleOrDefault(s => s.StandardId == searchResult.StandardCode.ToString());
```

Comment:
Is there scope to change StandardId to an int type?

Reply:
I tried to do this before, but after about 2 days, gave up and had to reel it back.  It's all over the place.

This is work  best done (if at all) when we go through the second phase of making standardCollation the only source of truth about standards inside assessor-service.  As all objects will be mapped into that.

Reply:
Fair comment :)